tests/periph_i2c/main.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2022 Mesh4all mesh4all.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @brief       I2C low level test
+ *
+ * @author      Luis A. Ruiz <luisan00@hotmail.com>
+ */
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+
+#include "periph_conf.h"
+#include "periph/gpio.h"
+#include "periph/i2c.h"
+
+#include "ztimer.h"
```

Comment:
I didn't find an use of ztimer in this test, Why does it need it?

Reply:
Yeah, good one